src/Auth0UserUpdateUtilities.js

```diff
@@ -0,0 +1,93 @@
+const ManagementClient = require("auth0").ManagementClient;
```

Comment:
We should pin this to a specific version, as it stands right now, this would default to `2.0.0-alpha-5` since when a version is not specified, we use whatever version shows up first [here](https://github.com/auth0/webtask-monorepo/blob/master/packages/webtask-modules/packages.auth0-node12.json#L270).

Reply:
So ... I'm still confused why we would pin in a public NPM module. No other require-able modules would pin their versions. And this could be included, theoretically, somewhere else and fail on that require statement.

Reply:
You're totally right! I confirmed with @jpadilla that this will resolve to whatever is the appropriate version when the webtask layer is built. 

Sorry for leading you astray.